app/routes/goal_routes.py

```diff
@@ -0,0 +1,91 @@
+from app import db
```

Comment:
👍 

This all looks good, but consider how the feedback I provided on Task could apply here as well.